### PR TITLE
Fixes Honk's terrible spelling

### PR DESCRIPTION
**Pull request by Pontenerd**
Don't let the Swedish touch spelling

## About The Pull Request

Fixes a spelling mistake @GoatMonarch made

## Why It's Good For The Game

Ansome won't cry himself to sleep knowing we have spelling mistakes

## Changelog
:cl:

spellcheck: fixed a typo

/:cl:

### CODE REVIEW

**Comment by GoatMonarch** [comment]
Word hard.
Clown sad...

File: code/game/objects/items/manuals.dm [hunk 1]
```diff
@@ -481,20 +481,20 @@
...
+
 	//Should only be used as maint loot / easter eggs
-/obj/item/book/manual/wiki/sop/munitions/puce 
+/obj/item/book/manual/wiki/sop/munitions/puce
 	icon_state = "sop_muni_puce"
 
 /obj/item/book/manual/wiki/sop/catalogue //Directs to standard SoP page. Able to check any department at will. For command / lawyer
-	name = "Standard operating Prosedure: Catalogue"
-	title = "Standard operating Prosedure: Catalogue"
+	name = "Standard operating Procedure: Catalogue"
```
Comment (Tennessee116) [inline code/game/objects/items/manuals.dm:496]:
```suggestion
	name = "Standard Operating Procedure: Catalogue"
```

File: code/game/objects/items/manuals.dm [hunk 2]
```diff
@@ -481,20 +481,20 @@
...
+	title = "Standard operating Procedure: Catalogue"
```
Comment (Tennessee116) [inline code/game/objects/items/manuals.dm:497]:
```suggestion
	title = "Standard Operating Procedure: Catalogue"
```

**Comment by Tennessee116** [comment]
The wiki refers to it as Standard Operating Procedure with a capital O.

**Comment by GoatMonarch** [comment]
> The wiki refers to it as Standard Operating Procedure with a capital O.

1. It's clearly SoP not SOP. Come on, it's the current year. 
2. The wiki is yet to have a Munitions SoP as a thing. The wiki has a say when it actually is worth listenin to.
3. This should have been mentioned in the original PR. And this is a hill I am willing to die on.

**Pull request merged**